style/style.css

```diff
@@ -26,4 +26,11 @@
 .pagination {
     margin-top: 0;
     margin-bottom: 0;
+}
+
+.users-items {
+    margin-top: -28px;
```

Comment:
I found that you used it to place pagination button and text in one line. Really bad and dirty hack.

I propose not keeping these items in one line. Text about a count of users set under the table and pagination button will have a place in a new line.
![joxi_screenshot_1504017774932](https://user-images.githubusercontent.com/20643403/29827033-8a8ad2b6-8ce1-11e7-99c5-2b039b5203f4.png)